### PR TITLE
section testimoni isoman

### PR DESCRIPTION
**Pull request by maulanayuseph**
**Task** 
**https://trello.com/c/78ai6R1H/881-web-s4-isoman-penambahan-section-testimoni-di-bawah-halaman**

**Preview**

**Link Preview**
https://deploy-preview-320--jabarprov-covid19.netlify.app/isoman

### CODE REVIEW

**Review by Ibwedagama — APPROVED** [review]

**Review by Arif9878 — APPROVED** [review]

**Review by naufalihsank — APPROVED** [review]

**Review by yoslie — APPROVED** [review]

**Review by bangunbagustapa — APPROVED** [review]

File: components/_pages/testimonial/testimoni.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+export default [
+  {
+    Name: 'Febby Wulandari',
+    Address: 'Jl Rawa Silam, Jabar',
+    Testimoni: 'Sangat membantu sekali bagi warga yang kurang mampu membeli vitamin dan obat-obatan.'
```
Comment (adrianpdm) [inline components/_pages/testimonial/testimoni.js:5]: is it possible to change keys of object using camelCase?

File: components/_pages/testimonial/TestimoniCard.vue [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        {{ body }}
+      </p>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    nama: {
```
Comment (adrianpdm) [inline components/_pages/testimonial/TestimoniCard.vue:35]: can these props be written in English?

Comment (maulanayuseph) [reply components/_pages/testimonial/TestimoniCard.vue:35]: thanks, will fix it asap

File: components/_pages/testimonial/testimoni.js (hunk 1 above)
Comment (maulanayuseph) [reply components/_pages/testimonial/testimoni.js]: thanks, already fixed on this commit [#09001c1](https://github.com/jabardigitalservice/pikobar-jabarprov-go-id/pull/320/commits/09001c11fb90021af663526487bead1fb10f6d13)

**Pull request merged**